File: src/main/java/com/github/graqr/threshr/model/redsky/products/summary/Grocery.java
package com.github.graqr.threshr.model.redsky.products.summary;

import com.fasterxml.jackson.annotation.JsonProperty;
import io.micronaut.serde.annotation.Serdeable;

@Serdeable
public record Grocery(@JsonProperty("is_active") Boolean isActive) {

}

File: src/main/kotlin/com/graqr/threshr/model/redsky/product/Grocery.kt
package com.graqr.threshr.model.redsky.product

import com.fasterxml.jackson.annotation.JsonProperty
import io.micronaut.serde.annotation.Serdeable

@Serdeable
data class Grocery(
    @param:JsonProperty("is_active") val isActive: Boolean
) {}
